Userland/DevTools/HackStudio/LanguageServers/Cpp/Tests/complete_includes.cpp

```diff
@@ -2,4 +2,3 @@
 #include <sys/cdef
 
 void foo() {}
-
```

Comment:
This PR still includes a bunch of unrelated accidental changes, that should not be part of this PR. 
Including here...

Reply:
that and other files from this default from the branch, I undo these files change from prev commit, because prev commit they all not get the right format cause clang automation formatter in my local device